Networking/Networking/Network/Network.swift

```diff
@@ -19,13 +19,6 @@ public protocol MultipartFormData {
 ///
 public protocol Network {
 
-    /// Designated Initializer.
-    ///
-    /// - Parameters:
-    ///     - credentials: WordPress.com Credentials.
-    ///
-    init(credentials: Credentials)
```

Comment:
This is removed to make `Network` reusable for other types. `Credentials` is now required only for `AlamofireNetwork`. In fact I think we may want to rename `AlamofireNetwork` to `JetpackTunneledNetwork`, but I'm not changing this in this PR.